src/features/events/components/EventParticipantsCard.tsx

```diff
@@ -31,7 +31,7 @@ const EventParticipantsCard: FC<EventParticipantsCardProps> = ({ model }) => {
   const eventData = model.getData().data;
   const messages = useMessages(messageIds);
   const reqParticipants = eventData?.num_participants_required ?? 0;
-  const availParticipants = eventData?.num_participants_available ?? 0;
```

Comment:
Perhaps we want to remove eventData.num_participants_available from the model as well so that this error is no longer possible to make?

Reply:
Maybe the platform API should be updated to exclude cancelled participants?

Reply:
Yes you're right!